src/libs/ReportActionsUtils.ts

```diff
@@ -265,7 +265,7 @@ function getContinuousReportActionChain(sortedReportActions: ReportAction[], id?
     if (id) {
         index = sortedReportActions.findIndex((obj) => obj.reportActionID === id);
     } else {
-        index = sortedReportActions.findIndex((obj) => obj.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
+        index = sortedReportActions.findIndex((obj) => obj.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD || obj.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED);
     }
 
     if (index === -1) {
```

Comment:
@situchan In your case, it will return an empty array since we don't find the report actions that is created action or pending action is not add.

Reply:
> It's bug. We should create report optimistically without skeleton but grayed out.

To cover all case above I think we can do my alternative solution which will return the current `sortedReportActions ` if the `id` is `empty` and `index` is `-1`

Reply:
@roryabraham what do you think?

And this case should also be added in unit test

Reply:
@situchan Updated the solution and unit test.